Hide anchor link on h1

bboulesteix opened the pull request:
This pull request updates our theme in order to close #22. It also hides the anchor link icon on h1 titles, because there's no difference between the link to a page and the link to the h1 of that said page—because we're only using one h1 per page.

## Preview

Note how the anchor icon is now disabled on hover. Sure, it's not perfect, since it's done using CSS, meaning that users can still get a h1 anchor link if they click on the exact location of the (hidden) anchor icon, but that's a decent start in my view.

| Before | After |
|---|---|
|![2022-10-14-Safari Technology Preview](https://user-images.githubusercontent.com/3286488/195810161-a822dafa-0a8b-46a2-8c1e-7a74c32549ce.png)|![2022-10-14-Safari Technology Preview 3](https://user-images.githubusercontent.com/3286488/195810201-a98330d5-e7c1-483f-9c2e-426925f84789.png)|

bboulesteix committed:
Update remote theme

This commit updates our Just the docs them to version 0.4.0.rc3.

bboulesteix committed:
Hide HTML anchor link on h1 headings

This commit hides the default anchor link provided by the theme on h1 headings. Since we only use one h1 heading per page, it makes little sense to offer a link for both our pages as well the h1 of our pages—because it boils down to the exact same thing, basically. Anchor links remain active on h2, h3, and so on…

bdelaforest commented on _sass/custom/custom.scss:
```diff
@@ -132,3 +132,12 @@ h4,
+  display: none;
+  svg {
+    visibility: hidden;
+  }
```
Does it work if you only set `display: none` here?

bdelaforest replied:
Actually, it doesn't work well if you keep `:hover` in the rule, But seems to work well if you do just this: 

```css
h1 > .anchor-heading {
    display: none;
}
```

bboulesteix replied:
> Does it work if you only set display: none here?

I tried that first, and it seemed to work, but the icon would briefly appear randomly anyway, so I figured: let's add as many rules as we can to make sure that the icon doesn't show up. But it's probably too much, you're right.

> Actually, it doesn't work well if you keep :hover in the rule.

I'll follow your advice and add your code snippet to the PR. Thanks. 🙏

bboulesteix committed:
Edit anchor-masking rule

This commit simplifies the rule used to prevent the h1 anchor icon from appearing on article pages. Thanks Benoit.

bdelaforest commented:
Closes #22
